### PR TITLE
Adds a demo multi-site scanning script

### PR DESCRIPTION
**Pull request by jwidavid**
See the README for testing instructions.

### CODE REVIEW

**Comment by ArSn** [comment]
I wonder if we should call this something like "multiple sites scanner" instead, just to avoid people confusing it with a [multisite](https://wordpress.org/documentation/article/create-a-network/).

**Comment by jwidavid** [comment]
> I wonder if we should call this something like "multiple sites scanner" instead, just to avoid people confusing it with a [multisite](https://wordpress.org/documentation/article/create-a-network/).

Yeah, I was wondering about that myself honestly. I just ran with the way it was described in the task, but if you also agree then I'm going to change it.

File: multiple-sites-scanner/multiple-sites-scanner.sh [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+	SERVER_NAME=$(_jq '.name')
+	SSH_CONNECTION=$(_jq '.ssh')
+	SITE_DIRECTORY=$(_jq '.site_dir')
+	SSH_KEY_PATH=$(_jq '.ssh_key_path')
+
+	# Run wp-cli commands to get list of plugins, themes, and WP version using SSH key-based authentication
+	WP_VERSION=$(ssh -i $SSH_KEY_PATH ssh $SSH_CONNECTION "wp core version --path=$SITE_DIRECTORY")
+	PLUGINS=$(ssh -i $SSH_KEY_PATH ssh $SSH_CONNECTION "wp plugin list --path=$SITE_DIRECTORY --fields=name,version --format=json")
+	THEMES=$(ssh -i $SSH_KEY_PATH ssh $SSH_CONNECTION "wp theme list --path=$SITE_DIRECTORY --fields=name,version --format=json")
```
Comment (ArSn) [inline multiple-sites-scanner/multiple-sites-scanner.sh]: I think these have an extra `ssh` in them for some reason.

```suggestion
	# Run wp-cli commands to get list of plugins, themes, and WP version using SSH key-based authentication
	WP_VERSION=$(ssh -i $SSH_KEY_PATH $SSH_CONNECTION "wp core version --path=$SITE_DIRECTORY")
	PLUGINS=$(ssh -i $SSH_KEY_PATH $SSH_CONNECTION "wp plugin list --path=$SITE_DIRECTORY --fields=name,version --format=json")
	THEMES=$(ssh -i $SSH_KEY_PATH $SSH_CONNECTION "wp theme list --path=$SITE_DIRECTORY --fields=name,version --format=json")
```

**Review by ArSn — DISMISSED** [review]
Works great except for these extra ssh commands. Also, conflict needs to be resolved before merging this.

**Review by bhoop — APPROVED** [review]
So merged, much wow 👍

**Pull request merged**